central/administration/events/datastore/telemetry.go

```diff
@@ -0,0 +1,88 @@
+package datastore
+
+import (
+	"context"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/administration/events"
+	"github.com/stackrox/rox/pkg/errorhelpers"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stackrox/rox/pkg/sac/resources"
+	"github.com/stackrox/rox/pkg/search"
+	"github.com/stackrox/rox/pkg/telemetry/phonehome"
+)
+
+// Gather the number of administrative events
```

Comment:
```suggestion
// Gather the number of administration events.
```

Reply:
This was a bit confusing, I searched for both in our code and there are more places where administration events is used, but still many places that use administrative events, strange. I've changed my usages to all be consistent with administration events

Reply:
Thanks! I checked the UI, and it says 'administration'.